libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer/importers/evm/index.ts

```diff
@@ -17,22 +18,23 @@ export type EVMToken = [
 ];
 
 export const importEVMTokens = async (outputDir: string) => {
-  try {
-    const supportedChainIds = [
-      1, 3, 5, 10, 25, 30, 40, 56, 57, 106, 137, 199, 246, 250, 420, 592, 1088, 1101, 1284, 1442,
-      8217, 8453, 42161, 84531, 421613, 245022934,
-    ];
+  console.log("Importing evm tokens...");
 
-    const chainNames: string[] = [];
+  const supportedChainIds = Object.values(cryptocurrenciesById)
+    .filter(cryptocurrency => cryptocurrency.ethereumLikeInfo?.chainId)
+    .map(cryptocurrency => cryptocurrency.ethereumLikeInfo!.chainId)
+    .sort((a, b) => a - b);
 
-    console.log("Importing evm tokens...");
+  const chainNames = new Map<number, string>();
 
-    for await (const chainId of supportedChainIds) {
+  for await (const chainId of supportedChainIds) {
+    try {
       console.log(`importing chain with chainId: ${chainId}...`);
       const erc20 = await fetchTokens<EVMToken[]>(`evm/${chainId}/erc20.json`);
```

Comment:
You can also use the `Promise.allSettled` here, so you can download all the chainId specific files at the same time 👌 
(Maybe just throw an error if the fetch error is anything else than a 404 error, as I don't know if you're not going to be rate limited by downloading that many files at the same time. Otherwise, we can just batch the downloads anyway 👍)